README.md

```diff
@@ -136,51 +144,73 @@ Scanner vendors are supposed to implement Scanner Adapters exposing the Scanner
          "id": "3fa85f64-5717-4562-b3fc-2c963f66afa6"
        }
        ```
-3. Try getting scan report (in unified Harbor format understandable by Harbor Web console):
+3. Try getting scan report (in the new v1.1 format understandable by Harbor Web console):
    ```
-   curl -H 'Accept: application/vnd.scanner.adapter.vuln.report.harbor+json; version=1.0' \
+   curl -H 'Accept: application/vnd.security.vulnerability.report; version=1.1' \
```

Comment:
Let's add new MIME type of `application/vnd.security.vulnerability.report; version=1.1]` in 1. where we give example of metadata reply

Reply:
Done.